hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/BaseEntityRepository.java

```diff
@@ -40,4 +41,17 @@
      */
     Optional<T> findById(I id);
 
+    /**
+     * Overrides
```

Comment:
please write why exactly you need this override and give a link to JpaRepository